content/en/blog/2022/otel-tuesday-v1-sunset.md

```diff
@@ -0,0 +1,28 @@
+---
+title: OpenTelemetry Tuesdays, Signing Off!
+date: 2022-03-21
+author: Austin Parker
+---
+
+When we started running OpenTelemetry Tuesday live streams back in 2019, the
+world was a lot different than it is today. Back then, it was very challenging
+for external participants and observers to understand what was going on with the
+overall status of the project. There weren't as many good resources for learning
+about OpenTelemetry, or for figuring out how to contribute.
+
+However, the project has matured by leaps and bounds since we started, and the
+associated rate of change and lack of discoverability that drove the stream as a
+concept has lessened. OpenTelemetry updates and news can be found in various
+[newsletters](https://o11y.news), [Twitter
+feeds](https://twitter.com/hashtag/opentelemetry),
+and a thriving community on [Slack](https://slack.cncf.io/)].
```

Comment:
There's a superfluous bracket at the end of this line.

Reply:
whoops good catch, thanks!